src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -1275,7 +1275,7 @@ public SolutionState WithDocumentText(DocumentId documentId, SourceText text, Pr
                 return this;
             }
 
-            return UpdateDocumentState(oldDocument.UpdateText(text, mode), textChanged: true);
+            return UpdateDocumentState(oldDocument.UpdateText(text, mode), contentChanged: true);
```

Comment:
this feels like a straight bug as well.  you could change the text, and we'd say `textChanged: true` but not pass `recomputeDependentVersions`...

Reply:
Yikes, yep that's not good.